mne/connectivity/tests/test_spectral.py

```diff
@@ -212,4 +214,41 @@ def test_spectral_connectivity(method, mode):
     assert (out_lens[0] == 10)
 
 
+@testing.requires_testing_data
+def test_epochs():
+    """Test spectral.spectral_connectivity with different data objects."""
+    data_dir = Path(__file__).parent.parent.parent / 'io' / 'tests' / 'data'
+    raw_fname = data_dir / 'test_raw.fif'
+    event_fname = data_dir / 'test-eve.fif'
+
+    raw = io.read_raw_fif(raw_fname)
+    events = read_events(event_fname)
+    picks = pick_types(raw.info, meg=True, eeg=True, stim=True, ecg=True,
+                       eog=True, include=['STI 014'], exclude='bads')
+
+    reject = dict(grad=1000e-12, mag=4e-12, eeg=80e-6, eog=150e-6)
```

Comment:
can this test be simplified at all?  E.g., not clear why you need the reject dict here (the testing data should be OK without it) and that makes the picks unnecessary too (after epoching you're restricting to only grads anyway).